src/main/java/codeu/model/util/Util.java

```diff
@@ -0,0 +1,17 @@
+package codeu.model.util;
+
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+public class Util {
+
+  /** Returns the date and time in the format:
+   *  Thu, 14 Jun 2018 05:01:09 GMT
+   */
+  public static String DateTimeFormatter(Instant creationTime){
```

Comment:
Maybe make this a verb phrase: `FormatDateTime` ?

Reply:
Alright!